kafka/util.py

```diff
@@ -64,3 +65,12 @@ class Dict(dict):
     See: https://docs.python.org/2/library/weakref.html
     """
     pass
+
+
+def get_client_factory(config: dict) -> Callable:
+    if 'client_factory' in config and config['client_factory'] is not None:
+        client_factory = config['client_factory']
+        assert callable(client_factory), f"'client_factory' should be a callable or None, is {type(client_factory)}"
```

Comment:
Older Python versions don't like the f-string (doesn't matter to us really but I guess it would be preferable if builds were passing).

Reply:
fixed